pages.es/common/!.md

```diff
@@ -0,0 +1,24 @@
+# Exclamation mark
+
+> Bash incorporado para sustituir con un comando encontrado en la historia.
+> Más información: <https://www.gnu.org/software/bash/manual/bash.html#Event-Designators>.
+
+- Sustituye con el comando anterior y lo ejecuta con sudo:
+
+`sudo !!`
+
+- Sustituye con un comando basado en un número de línea encontrado con `history`:
+
+`!{{número}}`
+
+- Sustituye con un comando que se utilizó un número especificado de líneas atrás:
+
+`!-{{número}}`
+
+- Sustituye por el comando más reciente que empiece por `cadema`:
```

Comment:
```suggestion
- Sustituye por el comando más reciente que empiece por cadema:
```
This shouldn't probably be marked as a command as it is a placeholder, will update it in other pages too.